doc/en/user/source/rest/api/styles.rst

```diff
@@ -94,7 +95,7 @@ Controls a given style.
    * - GET
      - Return style ``s``
      - 200
-     - SLD, HTML, XML, JSON
+     - SLD, HTML, XML, JSON, ZIP
      - HTML
```

Comment:
For now yes, only the sld. Changed to return the SLD instead. In the future can be improved to return a full package with all related resources if that is useful.